simple/sample/input/config.json

```diff
@@ -79,5 +79,6 @@
         "Provenance2 Name": "http://source1.com/provenance2"
       }
     }
-  }
+  },
+  "generateHierarchy": true
```

Comment:
groupStatVarByProperty? Since hierarchy is generated if MCF is provided too?

Reply:
I like that. Done.